public/js/projectsConstructor.js

```diff
@@ -1,49 +1,48 @@
 'use strict';
 
-var projects = [];
+var app = app || {};
 
-function Project (projectsListObj) {
-  this.title = projectsListObj.title;
-  this.url = projectsListObj.url;
-  this.githubURL = projectsListObj.githubURL;
-  this.img = projectsListObj.img;
-  this.skillsUsed = projectsListObj.skillsUsed;
-}
+var projects = [];
 
-Project.prototype.toHtml = function() {
-  var sourceHTML = $('#project-template').html();
-  var actualTemplate = Handlebars.compile(sourceHTML);
-  var newRawHTML = actualTemplate(this);
-  return newRawHTML;
-};
+(function(module) {
+  function Project (projectsListObj) {
+    this.title = projectsListObj.title;
+    this.url = projectsListObj.url;
+    this.githubURL = projectsListObj.githubURL;
+    this.img = projectsListObj.img;
+    this.skillsUsed = projectsListObj.skillsUsed;
+  }
 
-Project.loadAll = function (projectsList) {
-  projectsList.forEach(function(projectObject) {
-    projects.push(new Project(projectObject));
-  });
-};
+  Project.prototype.toHtml = function() {
+    var sourceHTML = $('#project-template').html();
+    var actualTemplate = Handlebars.compile(sourceHTML);
+    var newRawHTML = actualTemplate(this);
+    return newRawHTML;
+  };
 
-Project.initIndexPage = function() {
-  projects.forEach(function(project){
-    $('.projectsAnchor').append(project.toHtml());
-  });
-};
+  Project.loadAll = function (projectsList) {
+    projectsList.map ((projectObject) => {
+      projects.push(new Project(projectObject));
+    });
+  };
 
-$('#hamburgerMenu').on('click', function(event) {
-  event.preventDefault();
-  $('.navTab').toggle('fast');
-});
+  Project.initIndexPage = function() {
+    projects.map( project => $('.projectsAnchor').append(project.toHtml())
+    );
+  };
```

Comment:
You should probably leave this as a `forEach` since you are not creating a new array.

Reply:
switched back to a forEach!